pkg/service/restore/model.go

```diff
@@ -111,6 +112,14 @@ func (u *Unit) UnmarshalUDT(name string, info gocql.TypeInfo, data []byte) error
 	return gocql.Unmarshal(info, data, f.Addr().Interface())
 }
 
+func unitsContainTable(units []Unit, ks, tab string) bool {
+	idx := slices.IndexFunc(units, func(u Unit) bool { return u.Keyspace == ks })
+	if idx < 0 {
+		return false
+	}
+	return slices.ContainsFunc(units[idx].Tables, func(t Table) bool { return t.Table == tab })
+}
```

Comment:
This is the part assuring that view SSTables are skip ?
It checks if the keyspace.table is in units passed to restore.
View is not included into the units, right ?

Reply:
Units are stored in SM DB and are calculated only during the first run execution.
They don't contain the views, which are also stored in SM DB.